config/application.rb

```diff
@@ -8,9 +8,12 @@
 
 module TextCollections
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.0
```

Comment:
Any reason why we can't use `config.load_defaults 6.0`?

Reply:
Seems to run fine with 6.0 defaults, added a commit.